### PR TITLE
Vet Clinic Milestone #2: Query & update table

### PR DESCRIPTION
**Pull request by karlavdelgadof**
## Vet Clinic: Query and Update Table :question:  :hospital: :memo: :arrows_counterclockwise:  

These are the proposed changes :recycle:  for this project :arrow_down: :

- Added a new column `species` :frog: :lion:  of type string to animals table (Modified `schema.sql file.`) :+1: 
- Inserted the [new provided data](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Insert%20the%20following,escape%204%20times.) :card_file_box: :100: 
- Wrote specified transactions/queries :interrobang:  for this [task ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction,of%20your%20queries.) :+1: 
- Copied the SQL used in the previous points into their respective files in the generated repository. :open_file_folder: 

### Queries'/ Transactions Results: :camera: 

- **Transaction :hash: 1**
vet-clinic-q2-1

- **Transaction :hash: 2**
vet-clinic-q2-2

- **Transaction :hash: 3**
vet-clinic-q2-3

- **Queries :question:**
vet-clinic-q2-4

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -10,3 +11,77 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+-- TRANSACTION #1
+BEGIN;
+
+-- set set species column to unspecified
+UPDATE animals
+SET species = 'unspecified';
+
+-- roll back the transaction
+ROLLBACK;
+
```
Comment (sja-thedude) [inline queries.sql]: - [x] Kindly make sure to verify that change was made before ROLLBACK, and also verify that change was undone after ROLLBACK by adding this (before and after the rollback:
` SELECT species from animals;`

File: queries.sql [hunk 2]
```diff
@@ -10,3 +11,77 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- set the species column to pokemon for all animals that don't have species already set.
+UPDATE animals
+SET species = 'pokemon' WHERE species IS NULL;
+
+-- commit the transactions
+COMMIT;
+
```
Comment (sja-thedude) [inline queries.sql]: - [x] Kindly make sure to verify that change was made before COMMIT, and also verify that change persists after COMMIT by adding this (before and after the COMMIT:
` SELECT species from animals;`

File: queries.sql [hunk 3]
```diff
@@ -10,3 +11,77 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+-- TRANSACTION #3
+BEGIN;
+
+-- delete all records in the animals table
+DELETE FROM animals;
+
+-- roll back the transaction
+ROLLBACK;
+
+-- TRANSACTION #4
```
Comment (sja-thedude) [inline queries.sql]: - [x] Kindly make sure to verify that change was made before ROLLBACK, and also verify that change was undone after ROLLBACK by adding this (before and after the rollback:
`SELECT COUNT(*) FROM ANIMALS;`

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @karlavdelgadof, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct git-flow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (karlavdelgadof) [reply queries.sql]: @sja-thedude Thank you for pointing this out, however, I had already added these verification commands in the terminal, which are visible in the attached screenshots above :arrow_up:

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**